qcfractal/interface/models/rest_models.py

```diff
@@ -1122,3 +1140,115 @@ class QueueManagerPUTResponse(ProtoModel):
 
 
 register_model("queue_manager", "PUT", QueueManagerPUTBody, QueueManagerPUTResponse)
+
+
+## advanced procedures queries
+
+class OptimizationBestResultsBody(ProtoModel):
+    class Data(ProtoModel):
+        opt_ids: QueryObjectId = Schema(
```

Comment:
We usually try to write this out as `optimization_ids` to be verbose. What do you think?

Reply:
Yes, that's better